jobfunnel/config/cli.py

```diff
@@ -151,18 +151,28 @@ def parse_cli(args: List[str]) -> Dict[str, Any]:
     search_group.add_argument(
         '-ps',
         dest='search.province_or_state',
-        type=str,
+        type=None,
+        default=None,
         help='Province/state value for your job-search area of interest. '
              '(i.e. Ontario).',
-        required=True,
+        required=False,
     )
 
     search_group.add_argument(
         '-c',
         dest='search.city',
-        type=str,
+        type=None,
+        default=None,
         help='City/town value for job-search region (i.e. Waterloo).',
-        required=True,
+        required=False,
+    )
+
+    search_group.add_argument(
```

Comment:
I am a bit confused by the `remote-in-city` and `remote-in-country` arguments, why can't we simply provide a `--remote` argument?

Reply:
I just tried to make this part correspond 1:1 with the config.yaml. 
If a city is indicated and remote is set, then remote will override the search.

Reply:
I guess it doesn't make sense to me why we need this, since if a user provides a country but no city with --remote it would have the intended effect no?